.gitignore

```
@@ -62,3 +62,8 @@ temp/
 ._*
 .Trashes
 .svn
+
+############
+## Local overrides
+############
+.wp-env.override.json
```

```suggestion
############

.wp-env.override.json
```

Added line break per others in file.